errs.go

```diff
@@ -118,6 +148,12 @@ func (e *errs) As(target interface{}) bool {
 }
 
 func (e *errs) PrettyPrint() {
+	if e == nil {
```

Comment:
![badge](https://img.shields.io/badge/review-q-blue.svg)
This logic is not work. ( It's not a bug, but a fundamental problem that needs to be solved. )